.claude/skills/code-review/SKILL.md

```
@@ -77,11 +90,52 @@ coderabbit --prompt-only 2>&1
+`git diff --stat` の出力から変更ファイル一覧を取得し、**エージェントルーティングテーブル**と照合して起動するエージェントを決定する。
+
+#### 判定手順
+
+1. 変更ファイルのパスを一覧化する
+2. 各エージェントの起動条件パターンと照合する
+3. 一致したエージェントを起動リストに追加する（`code-quality-reviewer` は常に追加）
+4. 起動リストをユーザーに提示する
+
+**提示例**:
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)